Huion_Kamvas_GT221/index.md

```diff
@@ -0,0 +1,18 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: Kamvas GT221
+image: Huion_Kamvas_GT221
+report_rate: 233
+working_area:
+    width: 16
+    height: 9
+    resolution: 5080
+pen:
+    pressure_levels: 8192
+frame_controls: 10 buttons, 1 touch bar
```

Comment:
Wow, this is interesting. Would be good to see what `usbhid-dump` reports for that touch bar and the buttons after `uclogic-probe` was executed on the tablet.

Reply:
I am not sure why do you need the buttons? I see it has the `frame_button.txt`

Reply:
Right, I thought this was without the tablet enabled, but I see now that the reports are already right, which is good, thanks. It would be good to know what the user did there actually, though. I think I can see the 10 buttons being pressed, but I'm not sure which order was that. I think I can see the touch bar used, but I'm not sure which direction is which in the report, and what is the definite coordinate range. Do you still have contact with the user? Can you ask to make a new dump with a specific button press order, so we can find out, and another dump of the touch bar with clearly specified sequence of full-range movement, repeated several times so that we're sure we get the full range and the direction correctly? Better yet, if you could figure it all out yourself, so I can simply make a driver, and perhaps walk you through it :)

This has nothing to do with this PR, let's get this merged.

BTW, can you see the buttons and the touch bar yourself in the report?

Reply:
I take the information from this issue : https://github.com/DIGImend/digimend-kernel-drivers/issues/141. 
You mean you want me modify something in `usbhid-dump` source code to get a specific button, ...? 
I am not sure about these hexa number representing. I am going to read about it

Reply:
No, no need to modify usbhid-dump, just find out what the reports mean exactly :) What I write above is how you can do it, organizationally.

Reply:
Okay. But I need to totally understand the information in dump file first.

Reply:
Yeah, I would recommend experimenting with your own tablet, and reading at least the beginning of the USB HID specification.